### DISCUSSION TITLE
How do I embed Wistia videos in docusaurus docs?

### DISCUSSION THREAD

**Discussion by pranay01** [facebook/docusaurus#5923] [Q&A]
I have hosted videos on Wistia and want to embed them in our user docs to guide users. How can I do that?

**Comment by juzhiyuan** [comment]
![image](https://user-images.githubusercontent.com/2106987/141261496-37d20a3d-ae74-4905-a11e-f47014a47974.png)

I just registered the site, and could you please try using `oEmbed` with `iframe`?
